### PR TITLE
update basic I

### PR DESCRIPTION
**Pull request by pitviper6**
Hey everybody - here's my second attempt at breaking up the big OR lessons into smaller ones. This is a breakout of Basic-I.  It covers moving/removing columns and sort.  I added an exercise using the UCSD Guardian data.  Take a look and let me know what you think - I'm hoping I got the exercise syntax right so that it works with the new template!  If this passes the smell test, I'll continue breaking down the bigger lessons based on the format of this one.

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -48,6 +48,6 @@ There are also general and specialist tutorials about using OpenRefine available
 * [OpenRefine News (monthly round up of new blog posts, tutorials and other information)](http://openrefine.org/blog.html)
 
 ### Downloading the data
-You need to download [doaj-article-sample.csv](https://github.com/data-lessons/library-openrefine/raw/gh-pages/data/doaj-article-sample.csv), which is a csv file. Make a note of the location you save the file.
+You need to download [doaj-article-sample.csv](https://github.com/data-lessons/library-openrefine/raw/gh-pages/data/doaj-article-sample.csv), which is a csv file. Make a note of the location you save the file. If you want to work on the extra exercise, you need to download [UCSD_Guardian.csv](https://github.com/data-lessons/library-openrefine/raw/gh-pages/data/UCSD_Guardian.csv) as well.
```
Comment (ostephens) [inline setup.md]: Suggest you make this more directive - we don't need the 'if you want to'.
Also should be 'extra exercises' I think

**Comment by pitviper6** [comment]
Yep, if people think this approach works, I'll create two more lessons from the Basic I - one on facets and one on clustering.

@ostephens would a better approach be to create a separate episode for the self-directed exercise?

**Comment by ccronje** [comment]
Thanks @pitviper6. I agree with this breakout approach.

I think a hidden/drop-down for the self-directed exercise is better than creating a separate episode. What @mkuzak discussed in #41 & #44 (which merged into #43) made things clear for me. If I understand correctly - @ostephens's exercises become the walk through examples and do not require exercise labels. The self-directed exercises based on UCSD data become the hidden/drop-down examples.

So looking at a SC Shell example http://swcarpentry.github.io/shell-novice/04-pipefilter/ @ostephens instruction comes under the yellow sections and UCSD challenges are the orange/brown drop-downs.

**Comment by ostephens** [comment]
@ccronje just a +1 to that summary - that's my understanding as well

**Comment by pitviper6** [comment]
Yep, that's what I did here.  From what I can deduce, the ">>" and ">>>" are the indicators to the template that these are exercises/solutions and are branded by the orange drop downs.

Also, I'll change the label to 'extra exercises' and drop the 'if you want to'  YOU WANT TO! 

I'll continue to work on the breakouts for basics I, then move on to basics II.

**Comment by drjwbaker** [comment]
For info, just wanted to note that this template update https://github.com/data-lessons/library-openrefine/pull/47 Hopefully it won't mess with this ongoing PR!

**Comment by pitviper6** [comment]
I don't *think* it will - it's basic MD, and I took a look at the MD examples that used the new SWC template for the added Extra Examples. That's the only part that I'm a little concerned over, but I think it'll be ok!

**Comment by pitviper6** [comment]
Is there a way to test it with the new template without merging this PR?

**Comment by ostephens** [comment]
None of the template changes touch the files in this current PR - so I think should be OK

**Comment by gvwilson** [comment]
There's a rendered version in the gh-pages branch of my fork of the repo 
- you can view it online at https://gvwilson.github.io/library-openrefine/

**Comment by pitviper6** [comment]
Yep, I've seen those! I'm wondering how my changes here are going to render, since I'm breaking up the Basic I lesson into three smaller lessons (this is the first one covering columns and sorting) and adding extra exercises in the orange-colored drop-down format w/ solutions.

**Comment by gvwilson** [comment]
Hi Juliane,

I just merged your PR on my desktop and previewed it - there a couple of 
minor rendering glitches that I can fix up, so the only real issue is 
the renaming of episode files.  If you're finished making your changes, 
and then give me a list of which episode files you modified and which 
you created, I can re-label things.

Cheers,

Greg

**Comment by pitviper6** [comment]
Greg,

Thanks!  I'm hoping to get two more smaller episodes w/ extra exercises out of the Basics I lesson by the end of this week. I'll get you the list once I've done that!

**Comment by ccronje** [comment]
Hi @pitviper6 @ostephens how would you like to proceed with this PR?
- New formatting is being discussed here https://github.com/data-lessons/library-openrefine/pull/58
- Smaller episodes is being discussed here https://github.com/data-lessons/library-openrefine/issues/14
- Should the UCSD Guardian data be part of this mozsprint issue https://github.com/data-lessons/library-openrefine/issues/62?
What do you think?

**Comment by pitviper6** [comment]
Hey everybody, I completely dropped the ball on this!

1.  I would like to add the blockquotes formatting. Even if we did change
our views on what was an exercise and what was a challenge, the edit would
take minimal effort - you'd just change to name of the blockquote.

2. I'd love to continue doing what I'm doing with breaking the episodes
into smaller bits. I completely let this go when my job situation went
bonkers last fall, and I'm just now getting to the point where I can jump
back in and try to complete the breakdown.

3. I REALLY need to start from the beginning with the UCSD data and figure
out what I was trying to do with that. I'm not sure its really the best fit
for a digital humanities lesson so maybe I need to find another dataset....

Juliane.

**Comment by ccronje** [comment]
Thanks for your help @pitviper6! Will it help if I close this PR? After @jt14den's PR https://github.com/data-lessons/library-openrefine/pull/63 is merged it might be easier to go from there, what do you think?

**Comment by ostephens** [comment]
In favour of closing this PR
One #63 is merged think we need to apply same styling across all episodes before moving onto the lesson breakdown and alternative data sources (n.b. latter could overlap with #62 )

Think episode breakdown could work as part of mozsprint - especially alongside #61 and #38

**Comment by pitviper6** [comment]
Definitely, close that one! 


Sent from my Sprint Samsung Galaxy S7.
-------- Original message --------From: carmi cronje <notifications@github.com> Date: 5/14/17  4:09 AM  (GMT-05:00) To: data-lessons/library-openrefine <library-openrefine@noreply.github.com> Cc: Juliane Schneider <juliane.schneider@gmail.com>, Mention <mention@noreply.github.com> Subject: Re: [data-lessons/library-openrefine] update basic I (#46) 
Thanks for your help @pitviper6! Will it help if I close this PR? After @jt14den's PR #63 is merged it might be easier to go from there, what do you think?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/data-lessons/library-openrefine","title":"data-lessons/library-openrefine","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/data-lessons/library-openrefine"}},"updates":{"snippets":[{"icon":"PERSON","message":"@ccronje in #46: Thanks for your help @pitviper6! Will it help if I close this PR? After @jt14den's PR https://github.com/data-lessons/library-openrefine/pull/63 is merged it might be easier to go from there, what do you think?\r\n"}],"action":{"name":"View Pull Request","url":"https://github.com/data-lessons/library-openrefine/pull/46#issuecomment-301297497"}}}

**Pull request closed without merging**